feat: add default support for sphinx-llm (#590)
canonical/sphinx-docs-starter-pack: docs/reference/myst-syntax.md

---
relatedlinks: https://github.com/canonical/canonical-sphinx-extensions, [reStructuredText&#32;Primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html), [Canonical&#32;Documentation&#32;Style&#32;Guide](https://docs.ubuntu.com/styleguide/en)
myst:
  html_meta:

>     description: Find examples of the MyST syntax used in Canonical documentation files.

Reviewer: ```suggestion
    description: Reference for the MyST syntax conventions used by Canonical.
```
